instance/migrations/0004_add_django_popolo_people.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import re
+from urlparse import urlsplit, urlunsplit
+
+from django.db import migrations
+from django.contrib.contenttypes.management import update_contenttypes
+
+
+def is_proxy_url(url):
+    return url == 'everypolitician-writeinpublic.herokuapp.com'
+
+
+def update_source_url(original_url):
+    split_url = urlsplit(original_url)
+    has_popit_in_domain = 'popit' in split_url.netloc.split('.')
+    is_possible_api_url = re.match(r'/?$|^/api', split_url.path)
+    if is_proxy_url(split_url.netloc):
+        # If this is one of the faked PopIt instances for
+        # EveryPolitician, change that to the EveryPolitican JSON.
+        country, house = split_url.path.lstrip('/').split('/')
+        url = ('https://raw.githubusercontent.com'
+               '/everypolitician/everypolitician-data/master/data/'
+               '{country}/{house}/ep-popolo-v1.0.json').format(
+                   country=country, house=house)
+    elif has_popit_in_domain and is_possible_api_url:
+        # If looks like a PopIt instance, then replace that with a
+        # link to the PopIt instance's export.json:
+        split_as_list = list(split_url)
+        split_as_list[2] = '/api/v0.1/export.json'
+        split_as_list[3] = ''
+        split_as_list[4] = ''
+        url = urlunsplit(split_as_list)
+    else:
+        url = original_url
+    return url
+
+
+def forwards(apps, schema_editor):
```

Comment:
:smile: The scariest bit of this migration for me (although this whole PR is quite scary!) is actually the `update_source_url`, function above, since there are so many different PopIt URLs in the live database, lots of which don't work at the moment anyway. I hope that `update_source_url` will produce working URLs for those were working previously, but I think it's particularly worth double-checking that.